### PR TITLE
Language fixes/tasks for Feature/colpos

### PR DESCRIPTION
**Pull request by BenjaminBeck**

### CODE REVIEW

File: Classes/Form/Container/Grid.php [hunk 1]
```diff
@@ -41,6 +44,15 @@ public function build()
      */
     public function buildBackendLayoutArray(int $parentRecordUid)
     {
+
```
[the remark this answers is not in the document]
Comment (NamelessCoder) [reply Classes/Form/Container/Grid.php]: Or this should be called with the translated ID - if we still only need the ID.

**Comment by NamelessCoder** [comment]
I've cherry-picked `[TASK] Show only matching language in grid` from this set. I don't think we need the "set extension name" one, but will wait to see which LLL value you ended up with that caused problems - and I will do the refactoring required to pass a different parent UID to the buildBackendLayout methods :)

**Comment by NamelessCoder** [comment]
And I've pushed another change that calls the buildBackendLayout method with the `l18n_parent` of the record, if it is set.

That just leaves the LLL extension name question.

**Comment by BenjaminBeck** [comment]
For the LLL i get this exception: 2018-06-24 19_58_22-smec typo3 cms 8 7 13
Usualy i only need one language in the backend - so i write the labels direct into the template without LLL:... : ``<flux:grid.column colPos="0" name="Content" label="Content"/>``

**Comment by NamelessCoder** [comment]
Aaaah now I see the issue. You're using `label="column"` right? That is actually taken as a way to explicitly say "I want the label to be `column`" ;)

You can use `LLL:column` to indicate it should be translated. Translation (to first LLL:EXT: reference then by language service to LLL value) then happens directly in `getLabel()` - and creates the right values also for use in PageLayoutView column headers.

**Comment by NamelessCoder** [comment]
I'll add a check in Grid->buildBackendLayoutArray to prevent LLL on label values which do not start with LLL:EXT: - that should support custom labels that are not translated.

**Comment by BenjaminBeck** [comment]
Perfect, thanks!

**Comment by NamelessCoder** [comment]
Done - I will close this PR since all of those issues are now solved. Thanks for the patches!

**Pull request closed without merging**